src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -47,6 +48,13 @@ export function generatePerformanceQuery(location: Location): Readonly<NewQuery>
         : '-rpm';
   }
 
+  if (query?.query) {
+    const searchQuery = decodeScalar(query.query);
+    if (searchQuery) {
+      extra.query = `${PERFORMANCE_EVENT_VIEW.query} event.type:transaction transaction:"*${searchQuery}*"`;
```

Comment:
Why add a filter for transaction events? Errors can also have transactions.

Reply:
The performance view page only concerns itself with transaction events.

Reply:
Gotcha. Well this can still be removed because the PERFORMANCE_EVENT_VIEW constant already has this filter.

Reply:
Oops. Good point 👍

Reply:
Do you need to escape any quotes the user slams in.

Reply:
I've added double quote escaping here https://github.com/getsentry/sentry/pull/17763/commits/fc2f5fb78629a3da5490c93bcb86d474bf4dae20